python/pyspark/ml/tuning.py

```diff
@@ -847,9 +850,23 @@ def _fit(self, dataset: DataFrame) -> "CrossValidatorModel":
             subModels = [[None for j in range(numModels)] for i in range(nFolds)]
 
         datasets = self._kFold(dataset)
+
+        tmp_dfs_path = _get_temp_dfs_path()
+        spark_session = SparkSession.getActiveSession()
```

Comment:
in other issues, we found `SparkSession` operation maybe pretty slow

```suggestion
        spark_session = dataset._session
```